tools/diffacto/diffacto.xml

```diff
@@ -0,0 +1,230 @@
+<tool id="diffacto" name="Diffacto" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
+    <description>Comparative Protein Abundance from Covariation of Peptide Abundances</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.0.6</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">diffacto</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        diffacto -i '$input'
+        #if $db
+            -db '$db'
+        #end if
+        #if $samples
+            -samples '$samples'
+        #end if
+        -reference '$reference'
+        #if $normalize
+            -normalize $normalize
+        #end if
+        #if $farms_mu
+            -farms_mu $farms_mu
+        #end if
+        #if $farms_alpha
+            -farms_alpha $farms_alpha
+        #end if
+        -min_samples $min_samples
+        -impute_threshold $impute_threshold
+        -cutoff_weight $cutoff_weight
+        $use_unique
+        #if $scale == 'log2'
+            -log2 True
+        #else
+            -log2 False
+        #end if
+        $fast
+        -out '$output'
+        #if $mcfdr
+            -mc_out '$mc_out'
+        #end if
+        #if $loadings
+            -loadings_out '$loadings_out'
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="input" argument="-i" type="data" format="csv" label="Peptides abundances">
+            <help><![CDATA[
+                Peptides abundances in CSV format. 
+                The first row should contain names for all samples. 
+                The first column should contain unique peptide sequences. 
+                Missing values should be empty instead of zeros.
+            ]]></help>
+        </param>
+        <param argument="-db" type="data" format="fasta" label="Protein database" optional="true"
+               help="Optional if the peptide file has protein IDs in the second column"/> 
+        <param argument="-samples" type="data" format="tabular" label="Sample list" optional="true">
+            <help><![CDATA[
+               One run and its sample group per line, separated by tab. 
+               If None, read from peptide file headings, 
```

Comment:
which header?

Reply:
I improved the help text, so I hope that makes this more clear.